test/src/assets/scss/style.scss

```diff
@@ -0,0 +1,138 @@
+@charset 'UTF-8';
+
+/* ============================================================================
+   SCALLY TEST SUITE STYLESHEET
+   ========================================================================= */
+
+/**
+ * Styles for the Scally Test Suite.
+ *
+ * ----------------------------------------------------------------------------
+ *
+ * CONTENTS:
+ *
+ * 1. Scally Test Suite settings
+ * 2. Scally library
+ * 3. Scally Test Suite styles
+ *
+ * N.B. the above order of sections #1 and #2 must be maintained otherwise Sass
+ * will not compile!
+ */
+
+
+
+
+/* 1. Scally Test Suite settings incl. any Scally setting overrides
+   ========================================================================= */
+
+@import 'settings';
+
+
+
+
+/* 2. Scally library
+   ========================================================================= */
+
+/**
+ * Settings.
+ */
+
+@import '../../../../src/settings';
+
+
+/**
+ * Mixins/Functions.
+ */
+
+@function strip-unit($number) {
+    @if type-of($number) == "number" and not unitless($number) {
+        @return $number / ($number * 0 + 1);
+    }
+    @return $number;
+}
+@function rem($value) {
+    @if type-of($value) != "number" {
+        @error "You have to enter only a number for `$value`. You entered `#{$value}`.";
+    }
+    @return strip-unit($value) / strip-unit(16) * 1rem;
+}
+//@import '../../../../src/mixins-functions';
+
```

Comment:
I haven't done the **Mixins & Functions** yet but the Test site is using the new `rem()` function so I added the functions here temporarily.